test/unittest/unittest_nntrainer_internal.cpp

```diff
@@ -153,7 +152,7 @@ TEST(nntrainer_NeuralNetwork, init_07_p) {
   status = NN.setConfig("./test.ini");
   EXPECT_EQ(status, ML_ERROR_NONE);
   status = NN.init();
-  EXPECT_EQ(status, ML_ERROR_NONE);
+  EXPECT_EQ(status, ML_ERROR_INVALID_PARAMETER);
```

Comment:
@jijoongmoon please verify is this should fail or pass. Passing no `TrainData` is acceptable?

Reply:
Can you also clarify on `LabelData`, it's used by all train/valid/test it seems, That's a bit odd.

Reply:
1. If you mean the TrainData in ini file,  it's acceptable. We have two options to get training data. one is from file. and the other is from callback which defined by user.
2. Label Data keyword is used to calculate loss. Is there any other way to get the label data?

Reply:
Discussed off-line and make agreement.